addon/components/o-s-s/slider.stories.js

```diff
@@ -78,6 +78,16 @@ export default {
         defaultValue: { summary: '50' }
       }
     },
+
+    tooltiplabel: {
+      control: 'text',
+      description: 'Specific tooltip label for the slider handle',
+      defaultValue: `Value : ${defaultArgs.defaultValue}%, or apples if you want`,
```

Comment:
this is misleading ^^
should only be `defaultValue: ${defaultArgs.defaultValue}%`

Reply:
Cleaned up